op-program/compatibility-test/baseline-cannon-multithreaded-64.json

```diff
@@ -27776,5 +27776,4270 @@
     "impact": "This syscall is present in the program, but its execution depends on the actual runtime behavior. \n             If the execution path does not reach this syscall, it may not affect execution.",
     "reference": "https://github.com/ChainSafe/vm-compat?tab=readme-ov-file#how-it-works",
     "hash": "fe35df5511505818ce1133621afa9876c42fe27577b33044678438013b218eb2"
+  },
+  {
+    "callStack": {
```

Comment:
Added the new findings to the end to get a clear diff.  Will follow up to recreate the baseline in a separate PR and will drop the file, line and absPath values which are ignored by vm-compat anyway to make it easier to see the diff.

All of these findings already existed previously, we just have an additional path that winds up calling rowdy.NewStateFreezer.  It's always called with an empty path so actually just returns an in memory db but the analyzer can't tell that so also follows the path through `newResettableFreezer` which then triggers file related sys calls that aren't fully supported.